KPI/views/firefox_desktop_usage_forecast.view.lkml

```diff
@@ -112,12 +112,12 @@ view: insert_stmnt {
 
       SET row_count = (
         SELECT COUNT(*)
-        FROM mozdata.analysis.dou_forecasts
+        FROM moz-fx-data-shared-prod.telemetry.looker_forecasts_cache
         WHERE key = current_key);
 
       IF row_count = 0 THEN
 
-      INSERT INTO mozdata.analysis.dou_forecasts_cache
+      INSERT INTO moz-fx-data-shared-prod.telemetry.looker_forecasts_cache
```

Comment:
```suggestion
      INSERT INTO moz-fx-data-shared-prod.telemetry_derived.looker_forecasts_cache_v1
```

Reply:
I'm wondering if the service creds we use for this connection have write capabilities for shared-prod.